bindings/matrix-sdk-ffi/src/platform.rs

```diff
@@ -243,9 +243,23 @@ pub struct TracingConfiguration {
 }
 
 #[matrix_sdk_ffi_macros::export]
-pub fn setup_tracing(config: TracingConfiguration) {
+pub fn setup_tracing(mut config: TracingConfiguration) {
     log_panics();
 
+    // If there's no global directive, and a non-empty set of module/crate
+    // directives, then logs from modules and crates *not* appearing in the list
+    // of directives will be silently swallowed. This is bad, especially at the
+    // `error` level, for which we always want to see logs (notably because the
```

Comment:
The original reason for removing the global directive is to take **zero** chances that a 3rd party crate would log sensitive data, which would be hard to find and even harder to clean up.

Reply:
Yeah, that's a good point. I've implemented your suggestion to restrict this to only adding `panic=error` to the log filter line, since it's a more cautious choice.